src/lib.rs

```diff
@@ -264,8 +269,8 @@ fn search_references(
 }
 
 fn split_shorthand(shorthand: &str) -> (&str, &str) {
-    let parts = shorthand.split('/').collect::<Vec<&str>>();
-    (parts[0], parts[1])
+    let (first, rest) = shorthand.split_once('/').unwrap();
+    (first, rest)
```

Comment:
I sometimes create branches in the format of `feature/blah`, `feature/meh/ohno`, `upkeep/notagain`, etc.  I noticed that the split here might require the first item, and the rest of the string (basically `feature/blah` would become `feature` and so on).  This is why I did this change here; let me know if this is not ok.

Reply:
No the change is good. I forgot to handle this case.